src/utils/events/event-manager.js

```diff
@@ -149,3 +151,13 @@ export default class EventManager {
     return event => this.manager.emit(eventAlias, event);
   }
 }
+
+function ManagerMock(m) {
```

Comment:
JavaScripts "hoisting" of function names (allowing them to be used before declaration) is a mixed blessing. 

I'd move this up top, so that we can see how the mocking is done in one place.